ext/afform/core/ang/af/afRecaptcha.html

```diff
@@ -0,0 +1,5 @@
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<div class="crm-af-field">
+  <div class="g-recaptcha"></div>
```

Comment:
And then here you can insert it like this:
```suggestion
  <div class="g-recaptcha" data-sitekey="{{:: $ctrl.recaptchakey }}"></div>
```